milestone1Project/salad.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel= "stylesheet" href = "salad.css">
+    <title>Classic Greek Salad</title>
+</head>
+<body>
```

Comment:
Should include navbar on the recipe pages which link to the home page and the about page